src/deploy/standalone/noobaa_syslog.conf

```diff
@@ -4,6 +4,16 @@ $umask 0000
 $FileCreateMode 0644
 $EscapeControlCharactersOnReceive off
 
+$imjournalRatelimitInterval 0
+$imjournalRatelimitBurst 0
```

Comment:
why are these needed? can you add a comment to explain?

Reply:
Without these configuration rsyslog have a log rate limit, And because of this there is a chance that we might miss few logs with higher log levels. 
https://www.rsyslog.com/doc/configuration/modules/imjournal.html
_By default, ratelimiting is activated and permits to process 20,000 messages within 10 minutes,_
This is suggested in our doc also [Link](https://github.com/noobaa/noobaa-core/blob/0e9160dbf3c45e16a46dd3f3f8ff6aa0aa9e9d6b/docs/dev_guide/standalone_with_db.md#31-additional-syslog-configuration-for-rhel8)

Reply:
thanks for the info, but I am still not sure I understand - 
how is systemd journal related here?
aren't we sending log messages directly to syslog??

Reply:
@guymguym I am not sure about that

Reply:
so why should we add this?

Reply:
Previously in logs I could see message like `some messages are skipped due to rate limit`, Thats why I added this code.

Reply:
We need to understand this - perhaps this is because our process is sending log messages both to stdout and to syslog, and the systemd journal captures the stdout and then tries to use that also as input to rsyslog? are we even enabling imjournal?

I see from docs - https://www.rsyslog.com/doc/configuration/modules/imjournal.html
> Warning: Some versions of systemd journal have problems with database corruption, which leads to the journal to return the same data endlessly in a tight loop. This results in massive message duplication inside rsyslog probably resulting in a denial-of-service when the system resources get exhausted. This can be somewhat mitigated by using proper rate-limiters, but even then there are spikes of old data which are endlessly repeated. By default, ratelimiting is activated and permits to process 20,000 messages within 10 minutes, what should be well enough for most use cases. If insufficient, use the parameters described below to adjust the permitted volume. It is strongly recommended to use this plugin only if there is hard need to do so.

which makes me think that using this input is not something we want at all.
maybe we need to disable injournal?